src/shared/components/headerClipMaskLeft/headerClipMaskLeft.css

```diff
@@ -0,0 +1,7 @@
+.mask {
```

Comment:
instead of making these twice, just pull from an above `headerClipMask.css` since left and right styles are identical.

Reply:
Couldn't make do this exactly because they're identical, but refactored the CSS to be more DRY.